src/web/src/components/Posts/PostAvatar.tsx

```diff
@@ -58,11 +58,19 @@ const PostAvatar = ({ name, img, blog = '' }: AvatarProps) => {
       )
       .join('');
     return (
-      <a href={blog} className={classes.link}>
-        <Avatar className={classes.avatar}>
-          <p className={classes.text}>{initials}</p>
-        </Avatar>
-      </a>
+      <div>
+        {url != undefined && url?.length > 0 ? (
```

Comment:
This is really gonna sound nit-picky, but should be `!==` instead of `!=`

Reply:
Doing `url?` is the same as checking if `url` exists, so you don't need this.  However, @HyperTHD is right: you never want to use != or == in JS, since they both allow implicit type casting to happen, whereas !== and === will also check the types.